src/lib/utils.ts

```diff
@@ -4,3 +4,16 @@ import { twMerge } from "tailwind-merge";
 export function cn(...inputs: ClassValue[]) {
   return twMerge(clsx(inputs));
 }
+// this helper converts the numbers to any currency format for further use in project :)
+export function formatToCurrency(input:number, currencyType:string): string {
```

Comment:
instead of `string` use `"USD" | "Euro" | "Rupee"` type for `currencyType` parameter